package.json

```diff
@@ -65,6 +65,7 @@
     "big.js": "^6.1.1",
     "classnames": "^2.3.1",
     "colorthief": "^2.3.2",
+    "contains-path": "^1.0.0",
```

Comment:
Didn't add this for anything relevant to this PR, but continually got an internal package dep error that it couldn't find this so just added it as a dep to appease that. Happy to remove this and the updates to the package-lock if no one else was running into this issue and it was just something wrong with my local.

Reply:
That sounds odd 🤔  Haven't experienced this one so far.

Does this also happen after removing `node_modules` and doing an `npm install` again?

I'd really avoid adding random deps where possible.

Reply:
sounds good. I'll revert both of these. I think it was just an issue with my local